### ISSUE TITLE
[BUG][3.4.1823] Error in a fresh install

### ISSUE THREAD

**Issue by WalterLuis** [YetiForceCompany/YetiForceCRM#4004]
#### Actual Behavior

in "http://localhost/install/Install.php" I get this error:

```
Fatal error: require(): Failed opening required '/var/www/html/vendor/composer/../smarty/smarty/libs/bootstrap.php' (include_path='.:/usr/share/pear:/usr/share/php') in /var/www/html/vendor/composer/autoload_real.php on line 67
```

And the folder "[vendor](https://github.com/YetiForceCompany/YetiForceCRM/tree/developer/vendor)" does not exist the folder "smarty"

I installed both "smarty", "htmlpurifier" and other using the command composer "php compose.phar update", and I the installation process appears, but when I complete the installation process ... it stays in the window of "Installation in progress" until the end of time, and apache using the CPU apparently without errors.
![virtualbox_centos 7 - apache php mariadb](https://cloud.githubusercontent.com/assets/18419851/22886732/0b602822-f1f7-11e6-8924-ea6c6660dd16.png)

The database will be created from the installation process but not the tables :S
![virtualbox_centos 7 - apache php mariadb 2](https://cloud.githubusercontent.com/assets/18419851/22886698/e5078fd0-f1f6-11e6-8076-9166f6b704b7.png)

#### Your Environment

* YetiForce Version used: 3.4.1823
* Browser name and version: Firefox 45.7.0 for Linux
* Environment name and version: Apache/2.4.6 (CentOS), PHP 5.6.30, MariaDB 5.5.52
* Operating System and version: CentOS Linux release 7.3.1611

**Comment by WalterLuis** [comment]
Maybe ... could it be for this?
```
2017-02-13 14:12:47.5561 [warning] - Language file does not exist, module: Vtiger ,language: 
2017-02-13 13:12:47.5567 [warning] - Language file does not exist, module: Users ,language: 
2017-02-13 13:12:47.6123 [error] - Connection does not support reading schema information for '' DBMS. => /var/www/html/vendor/yii/db/Connection.php:746
2017-02-13 13:12:47.5152 [info][application] - 
$_SESSION = [
    '__HTTP_Session_Info' => 1
    'language' => ''
    'authenticated_user_id' => ''
    'layout' => ''
]====================================================================================================================================
2017-02-13 13:12:47.6227 [warning][yii\log\Dispatcher::dispatch] - Unable to send log via App\Log\Profiling: Not Supported: Connection does not support reading schema information for '' DBMS.
2017-02-13 13:12:47.5152 [info][application] - 
$_SESSION = [
    '__HTTP_Session_Info' => 1
    'language' => ''
    'authenticated_user_id' => ''
    'layout' => ''
]=========================================a===========================================================================================
2017-02-13 13:12:54.2373 [warning] - Language file does not exist, module:  ,language: en_us
2017-02-13 13:12:54.2026 [info][application] - 
$_POST = [
    'mode' => 'Step3'
    'lang' => 'en_us'
]
$_COOKIE = [
    'ITOP_SESSID' => '4dc706ce58a1b0cf86e7b'
]
$_SESSION = [
    '__HTTP_Session_Info' => 2
    'language' => ''
    'authenticated_user_id' => ''
    'layout' => ''
    'default_language' => 'en_us'
]====================================================================================================================================
2017-02-13 13:12:54.2388 [warning][yii\log\Dispatcher::dispatch] - Unable to send log via App\Log\Profiling: Not Supported: Connection does not support reading schema information for '' DBMS.
2017-02-13 13:12:54.2026 [info][application] - 
$_POST = [
    'mode' => 'Step3'
    'lang' => 'en_us'
]
$_COOKIE = [
    'ITOP_SESSID' => '4dc706ce58a1b0cf86e7b'
]
$_SESSION = [
    '__HTTP_Session_Info' => 2
    'language' => ''
    'authenticated_user_id' => ''
    'layout' => ''
    'default_language' => 'en_us'
]====================================================================================================================================
2017-02-13 13:17:03.2061 [warning] - Language file does not exist, module:  ,language: en_us
2017-02-13 13:17:03.1876 [info][application] - 
$_POST = [
    'mode' => 'Step3'
    'lang' => 'en_us'
]
$_COOKIE = [
    'ITOP_SESSID' => '4dc706ce58a1b0cf86e7b'
]
$_SESSION = [
    '__HTTP_Session_Info' => 2
    'language' => ''
    'authenticated_user_id' => ''
    'layout' => ''
    'default_language' => 'en_us'
]====================================================================================================================================
2017-02-13 13:17:03.2075 [warning][yii\log\Dispatcher::dispatch] - Unable to send log via App\Log\Profiling: Not Supported: Connection does not support reading schema information for '' DBMS.
2017-02-13 13:17:03.1876 [info][application] - 
$_POST = [
    'mode' => 'Step3'
    'lang' => 'en_us'
]
$_COOKIE = [
    'ITOP_SESSID' => '4dc706ce58a1b0cf86e7b'
]
$_SESSION = [
    '__HTTP_Session_Info' => 2
    'language' => ''
    'authenticated_user_id' => ''
    'layout' => ''
    'default_language' => 'en_us'
]====================================================================================================================================
2017-02-13 13:18:07.1742 [warning] - Language file does not exist, module:  ,language: en_us
2017-02-13 13:18:07.1184 [info][application] - 
$_POST = [
    'mode' => 'Step3'
    'lang' => 'en_us'
]
$_COOKIE = [
    'ITOP_SESSID' => '4dc706ce58a1b0cf86e7b'
]
$_SESSION = [
    '__HTTP_Session_Info' => 2
    'language' => ''
    'authenticated_user_id' => ''
    'layout' => ''
    'default_language' => 'en_us'
]====================================================================================================================================
2017-02-13 13:18:07.1763 [warning][yii\log\Dispatcher::dispatch] - Unable to send log via App\Log\Profiling: Not Supported: Connection does not support reading schema information for '' DBMS.
2017-02-13 13:18:07.1184 [info][application] - 
$_POST = [
    'mode' => 'Step3'
    'lang' => 'en_us'
]
$_COOKIE = [
    'ITOP_SESSID' => '4dc706ce58a1b0cf86e7b'
]
$_SESSION = [
    '__HTTP_Session_Info' => 2
    'language' => ''
    'authenticated_user_id' => ''
    'layout' => ''
    'default_language' => 'en_us'
]====================================================================================================================================

```

**Comment by PercyP** [comment]
Hi @WalterLuis 

I am having problems upgrading to the same release as your fresh install. I have same environment as you:

Environment name and version: Apache/2.4.6 (CentOS), PHP 5.6.30
Operating System and version: CentOS Linux release 7.3.1611

I also see this error in the sql log file:

[13-Feb-2017 16:17:31 +0000]: <55f1602c> [1] SELECT `vars`, `ip`, `changed`, now() AS ts FROM `roundcube_session` WHERE `sess_id` = '55f1602c6967156857be6ecc34be3750';
[13-Feb-2017 16:17:31 +0000]: <55f1602c> [2] UPDATE `roundcube_session` SET `changed` = now() WHERE `sess_id` = '55f1602c6967156857be6ecc34be3750';

And this repeatedly in the system log:

]2017-02-14 09:15:04.16 [error] - Incorrect mail access data: admin@mysite.com , INBOX
 - 0 modules/OSSMailScanner/models/Record.php (430)  >>  App\Log->error('Incorrect mail access data: admin@mysite.com , INBOX')
 - 1 modules/OSSMailScanner/cron/action.php (17)  >>  OSSMailScanner_Record_Model->executeCron('cgi-fcgi - Admin')
 - 2 cron/vtigercron.php (73)  >>  require_once('modules/OSSMailScanner/cron/action.php')
2017-02-14 09:15:04.42 [error] - Incorrect mail access data: admin@mysite.com , INBOX.Trash
 - 0 modules/OSSMailScanner/models/Record.php (430)  >>  App\Log->error('Incorrect mail access data: admin@mysite.com , INBOX.Trash')
 - 1 modules/OSSMailScanner/cron/action.php (17)  >>  OSSMailScanner_Record_Model->executeCron('cgi-fcgi - Admin')
 - 2 cron/vtigercron.php (73)  >>  require_once('modules/OSSMailScanner/cron/action.php')
2017-02-14 09:15:04.42 [error] - Incorrect mail access data: admin@mysite.com , INBOX.Sent
 - 0 modules/OSSMailScanner/models/Record.php (430)  >>  App\Log->error('Incorrect mail access data: admin@mysite.com , INBOX.Sent')
 - 1 modules/OSSMailScanner/cron/action.php (17)  >>  OSSMailScanner_Record_Model->executeCron('cgi-fcgi - Admin')
 - 2 cron/vtigercron.php (73)  >>  require_once('modules/OSSMailScanner/cron/action.php')
2017-02-14 09:15:04.7 [error] - Incorrect mail access data: admin@mysite.com , INBOX.spam
 - 0 modules/OSSMailScanner/models/Record.php (430)  >>  App\Log->error('Incorrect mail access data: admin@mysite.com , INBOX.spam')
 - 1 modules/OSSMailScanner/cron/action.php (17)  >>  OSSMailScanner_Record_Model->executeCron('cgi-fcgi - Admin')
 - 2 cron/vtigercron.php (73)  >>  require_once('modules/OSSMailScanner/cron/action.php')
2017-02-14 09:15:02.55 [info][application] - $_SESSION = [
    '__HTTP_Session_Info' => 1
    'authenticated_user_id' => ''
    'app_unique_key' => ''
]

See also: https://github.com/YetiForceCompany/YetiForceCRM/issues/3910#issuecomment-279420915
And https://github.com/YetiForceCompany/YetiForceCRM/issues/3996#issuecomment-279398574

**Comment by WalterLuis** [comment]
In version 4.0.0 it works

**Comment by PercyP** [comment]
@WalterLuis 

unfortunately it still doesnt work for me. Did you delete the updates folder or anything prior to running the update?

**Comment by WalterLuis** [comment]
@PercyP 

no, only use composer "php compose.phar update" from /var/www/html and I checked the php.ini configuration. 
Unfortunately I do not know if it was a change of mine in php.ini or if it was solved in some change the team of yetiforce, but I'm finishing automating in my spare time an installation from 0 on a CentOS 7 for the yetiforce [HERE](https://github.com/WalterLuis/server) of the steps I've done and I worked. I will try to finish it this Saturday and try it out for it to work.
